pom.xml

```diff
@@ -134,11 +146,33 @@
                 <groupId>org.jenkins-ci.tools</groupId>
                 <artifactId>maven-hpi-plugin</artifactId>
                 <configuration>
-                    <pluginFirstClassLoader>true</pluginFirstClassLoader>
                     <maskClasses>
-                        com.microsoft.azure.management.
                         com.google.common.
                         com.fasterxml.jackson.
+                        org.apache.commons.lang3.
```

Comment:
To resolve the class loader conflict when we deserialize the `TokenCredentialData`.

Currently, the deserialization logic uses the `org.apache.commons.lang3.SerializationUtils#deserialize`, which, at runtime, will be loaded by the `azure-commons` plugin without this mask. As a result, the deserialized object will be bounded to the `azure-commons` ClassLoader, and conflict with other Azure objects bounded to the `azure-vmss` plugin ClassLoader.